### ISSUE TITLE
Missing last change on "every N base" steps

### ISSUE THREAD

**Issue by tseemann** [iqbal-lab-org/simutator#3]
If i ask it to add a deletion every 1 million bases in Mtb 4.4 Mbp
It only adds 3 instead of 4
ie. at 1M, 2M, 3M but not at 4M
CC: @kristyhoran

**Comment by martinghunt** [comment]
Yes you're right. Am tempted to document this as a feature though. Unless you really need it fixed for SNPs.

I did it because if you're adding in complex variants that span several bp of the genome, then the last variant could be at the end of the genome. And then it tries to delete/mutate/whatever positions that are past the end of the genome. So I took the easy way out and made it stop 1 before that last variant it could add, even though it doesn't really need to for SNPs.
